janusgraph-hbase/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -540,8 +540,14 @@ public boolean exists() throws BackendException {
     public List<KeyRange> getLocalKeyPartition() throws BackendException {
         List<KeyRange> result = new LinkedList<>();
         try {
-            ensureTableExists(
-                tableName, getCfNameForStoreName(GraphDatabaseConfiguration.SYSTEM_PROPERTIES_STORE_NAME), 0);
+            if (skipSchemaCheck) {
+                logger.debug("Skipping schema check");
+                if (!exists()) throw new TemporaryBackendException("Table " + tableName + " doesn't exist in HBase!");
```

Comment:
This should be a `PermanentBackendException`. `TemporaryBackendException` is used when the unavailability is temporary.